packages/ai/src/generate-object/stream-object.test.ts

```diff
@@ -598,6 +598,106 @@ describe('streamObject', () => {
       });
     });
 
+    describe('options.onChunk', () => {
+      let result: Array<
+        Extract<
+          import('./stream-object-result').ObjectStreamPart<any>,
```

Comment:
can this import be extracted to the top of the file?

Reply:
done